assets/colors.md

```diff
@@ -0,0 +1,8 @@
+* WWW blue: #01093a
+* WWW brown: #46342f
+* WWW light brown: #eae3c5
+* ~WWW white: #ebe8cc~
+* ~WWW another white: #eae9cd~
+* ~WWW let's use another white cuz why not: #dcdbc1~
+* ~WWW omg why can't we decide on one white: #fffcdd~
+* WWW white but when there are 4 competing colors: #fef6c5 (inversion of WWW blue)
```

Comment:
[XD](https://xkcd.com/927/)

Reply:
yeah, this name was supposed to be a reference to that ;)